Source/Particles/Gather/GetExternalFields.H

```diff
@@ -54,8 +56,11 @@ struct GetExternalEBField
     const amrex::ParticleReal* AMREX_RESTRICT m_uy = nullptr;
     const amrex::ParticleReal* AMREX_RESTRICT m_uz = nullptr;
 
+    bool m_accelerator_lattice_defined = false;
+    LatticeElementFinderDevice d_lattice_element_finder;
```

Comment:
You can make this a single variable by using `std::optional<LatticeElementFinderDevice> d_lattice_element_finder` :)